tests/functional/export_v2/cram/minimal.t

```diff
@@ -2,11 +2,10 @@ Setup
 
   $ source "$TESTDIR"/_setup.sh
 
-Minimal export
+Minimal export -- single input (tree) and single output (dataset JSON)
 
   $ ${AUGUR} export v2 \
   >   --tree "$TESTDIR/../data/tree.nwk" \
-  >   --node-data "$TESTDIR/../data/div_node-data.json" \
   >   --output minimal.json
```

Comment:
This is a thing of beauty. Thank you for taking the time to make the dream a reality, @jameshadfield!